src/current/v23.2/show-virtual-cluster.md

```diff
@@ -0,0 +1,127 @@
+---
+title: SHOW VIRTUAL CLUSTER
+summary: The SHOW VIRTUAL CLUSTER statement provides details on virtual clusters and physical replication streams.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} The `SHOW VIRTUAL CLUSTER` statement lists the virtual clusters running in a CockroachDB cluster. `SHOW VIRTUAL CLUSTER` only supports inspecting virtual cluster status as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
+
+## Required privileges
+
+`SHOW VIRTUAL CLUSTER` requires the `admin` role or the `MANAGEVIRTUALCLUSTER` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM MANAGEVIRTUALCLUSTER TO user;
+~~~
+
+{% include enterprise-feature.md %}
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/physical-replication/show-virtual-cluster-diagram.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_spec` | The name of the virtual cluster.
+`REPLICATION STATUS` | Display the details of a replication stream.
```

Comment:
So in a single operation, you can show the status or the capabilities, but not both at once? (there is no `ALL` type shortcut?)

Reply:
Actually, thanks for mentioning this, you can do both as it turns out (just tested), so `SHOW VIRTUAL CLUSTER(S) WITH REPLICATION STATUS, CAPABILITIES`. I edited the diagram for this